src/main/proto/wfa/measurement/protocol/crypto/liquid_legions_v2_encryption_methods.proto

```diff
@@ -84,6 +84,20 @@ message PerBucketFrequencyDpNoiseBaseline {
   wfa.common.DifferentialPrivacyParams dp_params = 2;
 }
 
+// The request to complete work in the initialization phase.
+message CompleteInitializationPhaseRequest {
```

Comment:
Who makes this call to the mill? The herald?

Reply:
This is for the mill to call the C++ lib in JNI. There is no rpc going out.

Reply:
Ah ok thanks